standalone-metastore/metastore-server/src/main/resources/package.jdo

```diff
@@ -1523,6 +1523,29 @@
         <column name="LAST_UPDATE_TIME"/>
       </index>
    </class>
+    <class name="MReplicationMetrics" identity-type="application" table="REPLICATION_METRICS" detachable="true">
+      <field name="scheduledExecutionId" primary-key="true">
```

Comment:
I don't know if currently there is a way to remove these metrics entries - but it could be an option to piggy-pack on  the scheduled query execution removal by declaring a foreign key for this fields with "on delete cascade"